### ISSUE TITLE
Blazorise Cropper is throwing exceptions when cropping

### ISSUE THREAD

**Issue by njannink** [Megabit/Blazorise#4626]
I have embedded the new Blazorise Cropper and it's throwing loads of exceptions when updating the cropping area. Looks like one of the underlying eventhandlers is going crazy. Maybe you have an idea @stsrki 

```
blazorise.cropper.js?v=1.2.1.0:289 
        
       Error: System.Text.Json.JsonException: The JSON value could not be converted to System.Int32. Path: $ | LineNumber: 0 | BytePositionInLine: 17.
 ---> System.FormatException: Either the JSON value is not in a supported format, or is out of bounds for an Int32.
   at System.Text.Json.ThrowHelper.ThrowFormatException(NumericType numericType)
   at System.Text.Json.Utf8JsonReader.GetInt32()
   at System.Text.Json.Serialization.Converters.Int32Converter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, Int32& value)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   --- End of inner exception stack trace ---
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& state, Utf8JsonReader& reader, Exception ex)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=7.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCoreAsObject(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[Object](Utf8JsonReader& reader, JsonTypeInfo jsonTypeInfo, ReadStack& state)
   at System.Text.Json.JsonSerializer.Read[Object](Utf8JsonReader& reader, JsonTypeInfo jsonTypeInfo)
   at System.Text.Json.JsonSerializer.Deserialize(Utf8JsonReader& reader, Type returnType, JsonSerializerOptions options)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.ParseArguments(JSRuntime jsRuntime, String methodIdentifier, String arguments, Type[] parameterTypes)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.InvokeSynchronously(JSRuntime jsRuntime, DotNetInvocationInfo& callInfo, IDotNetObjectReference objectReference, String argsJson)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.BeginInvokeDotNet(JSRuntime jsRuntime, DotNetInvocationInfo invocationInfo, String argsJson)
    at Object.endInvokeDotNetFromJS (https://localhost:5000/_framework/blazor.webassembly.js?version=7.0.2:1:3549)
    at Object.Xt [as endInvokeDotNetFromJS] (https://localhost:5000/_framework/blazor.webassembly.js?version=7.0.2:1:63263)
    at Object.Gt [as invokeJSFromDotNet] (https://localhost:5000/_framework/blazor.webassembly.js?version=7.0.2:1:62760)
    at Object.Ii (https://localhost:5000/_framework/dotnet.7.0.2.hvhem7ofhx.js:5:71465)
    at _mono_wasm_invoke_js_blazor (https://localhost:5000/_framework/dotnet.7.0.2.hvhem7ofhx.js:14:103886)
    at wasm://wasm/00992bd6:wasm-function[313]:0x1d4d8
    at wasm://wasm/00992bd6:wasm-function[283]:0x1c906
    at wasm://wasm/00992bd6:wasm-function[221]:0xdff6
    at wasm://wasm/00992bd6:wasm-function[220]:0xce95
    at wasm://wasm/00992bd6:wasm-function[8115]:0x1a2195
```

**Comment by njannink** [comment]
You can also see these exceptions on the documentation page: https://blazorise.com/docs/extensions/cropper

```
Error: There was an exception invoking 'Crop'. For more details turn on detailed exceptions in 'CircuitOptions.DetailedErrors'
    at Ft.endInvokeDotNetFromJS (blazor.server.js:1:3718)
    at Ft._invokeClientMethod (blazor.server.js:1:75072)
    at Ft._processIncomingData (blazor.server.js:1:72696)
    at Ft.connection.onreceive (blazor.server.js:1:67009)
    at i.onmessage (blazor.server.js:1:51322)
```

**Comment by stsrki** [comment]
How can I reproduce the error on my side?

**Comment by njannink** [comment]
you can see the errors happening on the documentation page of blazorise aswell

**Comment by njannink** [comment]
I think one of the parameters of the Crop method in the adapter is undefined causing the javascript error

    public async ValueTask Crop( int startX, int startY, int endX, int endY )

**Comment by stsrki** [comment]
> you can see the errors happening on the documentation page of blazorise aswell

I'm trying it on the documentation, but I cannot reproduce it.

**Comment by njannink** [comment]
I found what it is. It tries to convert doubles to ints thats why the interop craches. PR is comming

**Comment by njannink** [comment]
@stsrki maybe browser (edge vs chrome), zoom levels or resolution causes this. But when I added some additional logging I could see the event details contained numbers that are doubles and not ints

**Comment by njannink** [comment]
Fixed with v1.2.2
